README.md

```diff
@@ -73,7 +73,10 @@ Please use `yarn` to manage the dependencies and scripts in this project. You sh
 
 The website is deployed using `now.sh`. The production website reflects the `master` branch. Pull requests which are made against master are considered staging branches.
 
-When contributing to the project branch from master and create a pull request, make sure that changes are approved and that your staging branch works as you expect. **When you merge into master the change will shortly be available on the production site.**
+1. When contributing to the project branch from master and create a pull request, `now.sh` bot will post a status of PR being deployed automatically.
+2. `now.sh` will generate a build and deploy in a temporary site which is available to get inside your pull request page.
```

Comment:
Maybe something like:

`now.sh` will generate a build and deploy to a temporary url which is available on your pull request page, along with the status of the build